.github/workflows/codeql.yaml

```diff
@@ -70,8 +70,9 @@ jobs:
       - name: Download go dependencies
         run: |
           go mod download
-          go mod download -modfile tools/go.mod
-          go mod download -modfile acceptance/go.mod
+          (cd tools && go mod download)
```

Comment:
Just for my own understanding, why not use `-modfile`?

Reply:
[This](https://github.com/enterprise-contract/ec-cli/actions/runs/5761947101/job/15620788857?pr=902#step:7:7) happens:
```
go: github.com/enterprise-contract/ec-cli/application@v0.0.0-00010101000000-000000000000 (replaced by ../application/): reading ../application/go.mod: open /home/runner/work/ec-cli/application/go.mod: no such file or directory
```